bundles/org.openhab.binding.prowl/src/main/resources/OH-INF/i18n/prowl_it.properties

```diff
@@ -30,5 +30,3 @@ pushNotificationActionEventLabel = Evento
 pushNotificationActionEventDescription = Nome evento.
 pushNotificationActionMessageLabel = Messaggio
 pushNotificationActionMessageDescription = Testo del messaggio.
-pushNotificationActionPriorityLabel = Priorit�
-pushNotificationActionPriorityDescription = Priorit� del messaggio (-2,-1,0,1,2).
```

Comment:
This should not be removed.
The problem was introduced by PR #12647 which added these two italian translations.

Reply:
I am going to upload the file in Crowdin.